dace/codegen/targets/rtl.py

```diff
@@ -436,82 +719,62 @@ def unparse_tasklet(self, sdfg: sdfg.SDFG, dfg: state.StateSubgraphView,
 V{name}* model = new V{name};
 
 // apply initial input values
-model->rst_i = 0;  // no reset
-model->clk_i = 0; // neg clock
-model->valid_i = 0; // not valid
-model->ready_i = 0; // not ready 
+model->ap_areset = 0;  // no reset
+model->ap_aclk = 0; // neg clock
+{valid_zeros}
+{ready_zeros}
 model->eval();
 
 {vector_init}
 
 // reset design
-model->rst_i = 1;
-model->clk_i = 1; // rising
+model->ap_areset = 1;
+model->ap_aclk = 1; // rising
 model->eval();
-model->clk_i = 0; // falling
+model->ap_aclk = 0; // falling
 model->eval();
-model->rst_i = 0;
-model->clk_i = 1; // rising
+model->ap_areset = 0;
+model->ap_aclk = 1; // rising
 model->eval();
-model->clk_i = 0; // falling
+model->ap_aclk = 0; // falling
 model->eval();
 
 // simulate until in_handshakes = out_handshakes = num_elements
-bool read_input_hs = false, write_output_hs = false;
-int in_ptr = 0, out_ptr = 0;
+{hs_flags}
+{in_ptrs}
+{out_ptrs}
 {num_elements}
 
-while (out_ptr < num_elements) {{
+// TODO tmp fix. Should have correct stopping condition
+//while (out_ptr_b < num_elements_a) {{
```

Comment:
Kill?

Reply:
Fixed in https://github.com/carljohnsen/dace/commit/07f72c298d6e90d24c2ee08b40fbe9a9083f463a